target/linux/ath79/dts/qca9533.dtsi

```diff
@@ -204,6 +213,12 @@
 
 };
 
+&cpuintc {
+	qca,ddr-wb-channel-interrupts = <2>, <3>, <4>, <5>;
+	qca,ddr-wb-channels = <&ddr_ctrl 4>, <&ddr_ctrl 2>,
```

Comment:
Please remove <2> from "qca,ddr-wb-channel-interrupts" and <&ddr_ctrl 4> from "qca,ddr-wb-channels" from cpuintc node because this is already handled on intc2 node.